ruby/.rubocop.yml

```
@@ -134,6 +134,8 @@ Naming/RescuedExceptionsVariableName:
 
 Rails:
   Enabled: true
+  Exclude:
+    - db/schema.rb
```

Maybe its worth adding as well

```suggestion
    - db/schema.rb
    - db/insights2_schema.rb
```